File: URI/C/NumerosPositivos.c
#include <stdio.h>

int main()
{
    float num, soma = 0; 
    int qte = 0;
    scanf("%f", &num);
    if(num >0) qte++, soma+=num;
    scanf("%f", &num);
    if(num >0) qte++, soma+=num;
    scanf("%f", &num);
    if(num >0) qte++, soma+=num;
    scanf("%f", &num);
    if(num >0) qte++, soma+=num;
    scanf("%f", &num);
    if(num >0) qte++, soma+=num;
    scanf("%f", &num);
    if(num >0) qte++, soma+=num;
    printf("%d valores positivos\n", qte);
    printf("%.1f\n", soma/qte);

    
    return 0;
}

File: URI/Java/NumerosPositivos.java
import java.util.Scanner;
import java.lang.Math;

public class NumerosPositivos {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        float num;
        int pos = 0;
        for (int i = 0; i < 6; i++) {
            num = input.nextFloat();
            if (num > 0) {
                pos++;
            }
        }
        System.out.println(pos+" valores positivos");
        input.close();
    }
}